File: javascript/1065.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8');
var lines = input.split('\n');
var arr = [parseInt(lines.shift()), parseInt(lines.shift()), parseInt(lines.shift()), parseInt(lines.shift()), parseInt(lines.shift())];
var r = 0;
arr.forEach(e => {
     if(e%2 === 0) {r++};
})
console.log(r + " valores pares")

File: C++/1065.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
 
    int a,r;
    
    for(int i=0;i <5;i++){
        cin >> a;
        if(a%2 == 0){
            r++;
        }
    }
    r--;
    cout << r << " valores pares" << endl;
    return 0;
}